docs/reference-guide/streams/transformation-modules/overview.md

```diff
@@ -89,42 +89,6 @@ The file is now inside your Docker container.
 
 </details>
 
-:::caution
```

Comment:
Why did you remove this? @vpavicic

Reply:
It was accidentally duplicated 
![image](https://user-images.githubusercontent.com/95473291/157181218-68a4ea12-a093-46d3-9e47-07c1572f1335.png)